js/sound-player.js

```diff
@@ -11,3 +12,24 @@ function handleSoundItemsClick(event) {
 }
 
 soundItems.forEach(item => item.addEventListener('click', handleSoundItemsClick));
+
+function handlePlayButtonClick(event) {
```

Comment:
you know...for someone who 'doesnt know javascript', you seem to know a lot about javascript ;)